linodecli/completion.py

```diff
@@ -47,11 +51,17 @@ def get_fish_completions(ops):
     completion_template = Template(
         """# This is a generated file by Linode-CLI! Do not modify!
 complete -c linode-cli -n "not __fish_seen_subcommand_from $subcommands" -x -a '$subcommands --help'
+complete -c linode -n "not __fish_seen_subcommand_from $subcommands" -x -a '$subcommands --help'
```

Comment:
The corresponding unit tests may need to be adjusted to expect these as a part of the output